### DISCUSSION TITLE
How to delete plugins from `ya pack`?

### DISCUSSION THREAD

**Discussion by Jaehaks** [sxyazi/yazi#1558] [1. Q&A] [answered]
`ya pack` has feature of adding plugin and installing. but it doesn't have deleteting plugin. 
Of course, I can delete forcely using `rm` but it doesn't delete the plugin from the list with `ya pack -l`
So When I use `ya pack -i` , the deleted plugin is installed.
how to delete plugin from the list?

**Comment by AnirudhG07** [comment, marked as answer]
You have to remove the plugin from `package.toml` file. It contains the list of all plugins and their recent commits.
It would be present in the same directory as your yazi.toml or `plugins` folder. So along with removing it from `plugins`, remove it from there too

**Reply by Jaehaks** [reply]
@AnirudhG07 Thanks:)

**Comment by sxyazi** [comment]
`ya pack` is now supported deleting packages (https://github.com/sxyazi/yazi/pull/2181), thanks to @MrAsler 🎉
